src/Page.js

```diff
@@ -827,10 +802,13 @@ Page.prototype.generate = function (builtFiles) {
  */
 Page.prototype.preRender = function (content) {
   let preRenderedContent = content;
+  const pageConfig = {
+  };
```

Comment:
The selection of config attributes to be given to plugins seems to be a bit too aribitrary... 

To be consistent with `postRender` below, maybe also include the `headingIndexingLevel`? Additionally, you can also include other search parameters such as `enableSearch` and `searchable`.

Finally, plugins may be interested in knowing the paths of the original source. `rootPath` and `sourcePath` can also be included.

These inclusions should also be made in `postRender`.

Reply:
Ahh I was thnking we could leave it blank first, then include whatever properties we may need when they show up

But sure, I can see how this is a little odd